test/blueprint/entity-blueprint.spec.js

```diff
@@ -99,6 +114,14 @@ describe('JHipster entity generator with blueprint', () => {
             it('contains the specific change added by the blueprint', () => {
                 assert.fileContent(`${CLIENT_MAIN_SRC_DIR}i18n/en/foo.json`, /awesomeAngularAppName/);
             });
+
+            // Verify if the custom entity config is persisted.
+            it('contains the specific config added', () => {
+                assert.fileContent('.jhipster/Foo.json', /customPreConfigValue/);
```

Comment:
```suggestion
                assert.fileContent('.jhipster/Foo.json', /"customPreConfigKey": "customPreConfigValue"/);
```
Can yo make the assertion more precise? Indeed `customPreConfigValue` is used for 2 different keys, so this might be confusing